### ISSUE TITLE
Add support for disconnect/reconnect via CLI resp. signals

### ISSUE THREAD

**Issue by the0ne** [phako/Sellerie#12]
Hi @phako first of all I'd like to say it's fantastic you refresh gtk-term which I use a lot =)

I always wished to have a command-line interface for gtk-term so that I could - from a script - disconnect, flash the usb-device and then reconnect the terminal.
I thought about using signals USR1 (connect) and USR2 (disconnect) but never found the time to start.

What do you think about this proposal?

**Comment by phako** [comment]
Yes but no. Yes, sounds nice, no, no UNIX signals. I think it is easier (and cleaner) to add actions to the GApplication that will then be remotely callable using the app (That's just standard GLib stuff so basically for free). But that needs some further clean-up of the app, i.e. single instance, multiple windows

Btw. There is also https://github.com/Jeija/gtkterm which is a bit less intrusive

**Comment by the0ne** [comment]
Added to gtkterm as https://github.com/Jeija/gtkterm/commit/2511ec23a060c78540593913034a8027b9482cdb
